SoftwareGuide/Latex/Architecture/SpatialObjects.tex

```diff
@@ -156,31 +147,23 @@ \subsection{SurfaceSpatialObject}
 
 \subsection{TubeSpatialObject}
 
-\doxygen{TubeSpatialObject} represents a base class for the representation
-of tubular structures using SpatialObjects. The classes \doxygen{VesselTubeSpatialObject} and
-\doxygen{DTITubeSpatialObject} derive from this base class.
-VesselTubeSpatialObject represents blood vessels extracted for an image and
-DTITubeSpatialObject is used to represent fiber tracts from diffusion tensor images.
+\doxygen{TubeSpatialObject} is a class for the representation
+of tubular structures using SpatialObjects.  In particular, it is intended
+to be used to represent vascular networks extracted from 2D and 3D images. 
+It can also be used to represent airways, nerves, bile ducts, and more.
+
+ The class \doxygen{DTITubeSpatialObject} is derive from this class
```

Comment:
(...) is derive : (...) is derived (...)

Reply:
Good catch @jhlegarreta ... adding this in the ExternalProject push.

Reply:
Fixed!   Thanks!